gallery_dl/extractor/mastodon.py

```diff
@@ -44,6 +44,11 @@ def items(self):
             del status["media_attachments"]
 
             status["instance"] = self.instance
+            if "@" in status["account"]["acct"]:
+                status["remote_instance"] = status["account"]["acct"].split('@')[1]
+            else: 
+                status["remote_instance"] = None
```

Comment:
```suggestion
            acct = status["account"]["acct"]
            status["remote_instance"] = \
                acct.rpartition("@")[2] if "@" in acct else None
```

- Accessing `status["account"]["acct"]` multiple times is slower than storing it in a local variable
- I prefer (r)partition over split. It's usually even faster.

Reply:
Thanks for you suggestions. I had messed up flake8 with GH GUI.